presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-impersonation/hive.properties

```diff
@@ -12,13 +12,13 @@ hive.metastore-cache-ttl=0s
 
 hive.metastore.authentication.type=KERBEROS
 hive.metastore.service.principal=hive/hadoop-master@LABS.TERADATA.COM
-hive.metastore.client.principal=hive/hadoop-master@LABS.TERADATA.COM
-hive.metastore.client.keytab=/etc/hive/conf/hive.keytab
+hive.metastore.client.principal=hive/_HOST@LABS.TERADATA.COM
```

Comment:
Hive metastore impersonates HDFS operations to this user. So, it must have permissions to do everything withing `/user/hive/warehouse` directory. Which in our case belongs to `hive`